src/drawing/backend_impl/bitmap.rs

```diff
@@ -165,6 +175,27 @@ impl<'a> BitMapBackend<'a> {
         }
     }
 
+    /// Helper function to convert with_buffer output
```

Comment:
It's useful for sure, but I have a little bit worried about adding this to Plotters.  As you may realized Plotters is a graphing library and any pixel format conversion should be out-of-scope topic in my opinion.  

I actually think changing the pixel format should be a functionality of image manipulation crate. So what you may want to do can be done with the following code as well https://docs.rs/image/0.22.3/image/enum.DynamicImage.html. 

Plotters actually uses this crate internally. To make the conversion convenient, I think exposing an API to borrow the internal image object is better than this function.

Reply:
> Plotters is a graphing library and any pixel format conversion should be out-of-scope topic in my opinion.

Fair enough. As for exposing the internal image, I'm not sure I follow. Do you mean that the  BitMapBacked should expose its internal state? Would it not be better to allow for different write formats at that point?
ex.
```
BitMapBackend::with_buffer(buffer, (width, height), <pixels as triples of u8s/pixels as single u32s/whatever else>)
```